lib/extractStyles.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
 var CSSDisplayNames = require('./CSSDisplayNames');
-var CSSPropertyOperations = require('react/lib/CSSPropertyOperations');
+var explodePseudoStyles = require('./explodePseudoStyles');
```

Comment:
Got it. Sorry about that. What makes something CamelCased vs. lowercase? Is your styleguide public?